tests/unit_tests/c_wrappers/res/enkf/test_simulation_batch.py

```diff
@@ -1,72 +0,0 @@
-import pytest
-
-from ert._c_wrappers.enkf import EnKFMain
-from ert._c_wrappers.enkf.config.ext_param_config import ExtParamConfig
-from ert._c_wrappers.enkf.config.gen_data_config import GenDataConfig
-from ert._c_wrappers.enkf.enums import RealizationStateEnum
-from ert.simulator.simulation_context import _run_forward_model
-
-
-@pytest.mark.integration_test
-def test_run_simulation_batch(setup_case, prior_ensemble):
```

Comment:
Why is the test removed?

Reply:
Because of this comment:
```
    # Observe that a significant amount of hardcoding
    # regarding the GEN_DATA and EXT_PARAM nodes is assumed
    # between this test, the config file and the forward model.
 ```

when a lot of that adding of nodes after the fact is no longer possible this test no longer passes.